### DISCUSSION TITLE
v5 icdar训练精度很低

### DISCUSSION THREAD

**Discussion by cqray1990** [PaddlePaddle/PaddleOCR#16436] [Q&A]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

训练好几遍，参数也改了，精度都上不去
[2025/05/31 01:52:51] ppocr INFO: best metric, hmean: 0.5987933634992458, is_float16: False, precision: 0.6265123619147817, recall: 0.5734232065479057, fps: 32.07059484882953, best_epoch: 844
I0531 01:52:52.153658 1482578 process_group_nccl.cc:159] ProcessGroupNCCL destruct 

### 🏃‍♂️ Environment (运行环境)

paddleocr3.0

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

```yaml
Global:
  model_name: PP-OCRv5_server_det
  debug: false
  use_gpu: true
  epoch_num: 1000
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/PP-OCRv5_server_det_icdar
  save_epoch_step: 30
  eval_batch_step:
  - 0
  - 1500
  cal_metric_during_train: false
  checkpoints: null
  pretrained_model:pretrain_models/PPHGNetV2_B4_ocr_det.pdparams
  save_inference_dir: null
  use_visualdl: false
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./checkpoints/det_db/predicts_db.txt
  distributed: true
Architecture:
  model_type: det
  algorithm: DB
  Transform: null
  Backbone:
    name: PPHGNetV2_B4
    det: true
  Neck:
    name: LKPAN
    out_channels: 256
    intracl: true
  Head:
    name: PFHeadLocal
    k: 50
    mode: large
Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 2
  regularizer:
    name: L2
    factor: 1e-6
PostProcess:
  name: DBPostProcess
  thresh: 0.3
  box_thresh: 0.6
  max_candidates: 1000
  unclip_ratio: 1.5
Metric:
  name: DetMetric
  main_indicator: hmean
Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/icdar2015/text_localization/
    label_file_list:
    - icdar/train_icdar2015_label.txt
    ratio_list:
    - 1.0
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - CopyPaste: null
    - IaaAugment:
        augmenter_args:
        - type: Fliplr
          args:
            p: 0.5
        - type: Affine
          args:
            rotate:
            - -10
            - 10
        - type: Resize
          args:
            size:
            - 0.5
            - 3
    - EastRandomCropData:
        size:
        - 640
        - 640
        max_tries: 50
        keep_ratio: true
        p: 0
    - MakeBorderMap:
        shrink_ratio: 0.4
        thresh_min: 0.3
        thresh_max: 0.7
        total_epoch: 1000
    - MakeShrinkMap:
        shrink_ratio: 0.4
        min_text_size: 8
        total_epoch: 1000
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - threshold_map
        - threshold_mask
        - shrink_map
        - shrink_mask
  loader:
    shuffle: true
    drop_last: false
    batch_size_per_card: 32
    num_workers: 8
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/icdar2015/text_localization/
    label_file_list:
    -icdar/test_icdar2015_label.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - DetResizeForTest:
        image_shape:
        - 736
        - 1280
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - shape
        - polys
        - ignore_tags
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 1
    num_workers: 2
profiler_options: null
```

**Comment by leo-q8** [comment]
有用 v5 的 官方检测模型直接评估过吗？
也请提供一下训练集和评估集数据连接，我们复现一下

**Comment by cqray1990** [comment]
> 有用 v5 的 官方检测模型直接评估过吗？ 也请提供一下训练集和评估集数据连接，我们复现一下

@leo-q8 直接用官方模型也测试国，同样很低 数据就是icdar 2015，

链接: https://pan.baidu.com/s/1o0vj_AT97E7_nJk8HzN10g 提取码: t8bv

@cuicheng01 @leo-q8 @jzhang533 @kerneltravel

**Comment by leo-q8** [comment]
@cqray1990 您好，数据连接好像过期了，能再提供一下吗

**Comment by cqray1990** [comment]
> [@cqray1990](https://github.com/cqray1990) 您好，数据连接好像过期了，能再提供一下吗

@leo-q8 链接: https://pan.baidu.com/s/1OPJ7btXXvzROac1NYs5L0Q 提取码: kvjr

**Comment by leo-q8** [comment]
@cqray1990 收到，我们验证一下

**Comment by cqray1990** [comment]
> [@cqray1990](https://github.com/cqray1990) 收到，我们验证一下

@leo-q8 验证结果怎么样了呢

**Comment by feiyilicare** [comment]
我尝试了下训练，精度指标还行，但检测结果边框会往里缩，4边都往里缩,数据是LSVT

**Reply by julesarts** [reply]
u can change the argument which name is unclip_ratio to 2 or 2.5.

**Reply by cqray1990** [reply]
> 我尝试了下训练，精度指标还行，但检测结果边框会往里缩，4边都往里缩,数据是LSVT

icdar的数据训练也正常？，直接用测试icdar数据精度很低
@feiyilicare
